yaml-tests/src/main/java/com/apple/foundationdb/relational/yamltests/YamlTestExtension.java

```diff
@@ -66,23 +72,29 @@ public void beforeAll(final ExtensionContext context) throws Exception {
                     new MultiServerConfig(1, 1115, 1116),
                     new ForceContinuations(new MultiServerConfig(1, 1117, 1118))
             );
+            maintainConfigs = List.of(
+                    new CorrectExplains(new EmbeddedConfig()),
+                    new CorrectMetrics(new EmbeddedConfig()),
+                    new CorrectExplainsAndMetrics(new EmbeddedConfig())
```

Comment:
Do we need all three of these here? Or would

Reply:
As it turns out, whenever you add a new query and you kind of just want it to put the explain into the yaml file, you need the option to correct both at the same time, since either one would fail if run separately, i.e. `CorrectMetrics` would fail due to the missing explain, and `CorrectExplain` would fail on the missing metrics.

Reply:
Why not just have one (just `CorrectExplainAndMetrics`)?

Reply:
probably you are right but sometimes you really don't want to change the explain if you know only the counters change. It is easier to make a mistake if you always ask for both although in reality that is probably going to happen anyways.

Reply:
Maybe I'm misunderstanding how this works, but won't this run all three of these if you set the tag on the test/test class? I think you'd need separate annotations for each if you wanted to just correct the counters, say